karate-tests/src/test/java/patients/healthcareWorker/authentication.feature

```diff
@@ -0,0 +1,85 @@
+@ignore
+Feature: Authentication for Healthcare Worker
+    
+  Reusable feature for authentication. When calling this feature,
+  you can optionally pass in a userID value to authenticate as a specific
+  test user. Otherwise, we default to shirley.bryne@nhs.net (656005750107)
+
+  For reference some valid cis2 mock usernames are:
+    - 656005750104 	surekha.kommareddy@nhs.net
+    - 656005750105 	darren.mcdrew@nhs.net
+    - 656005750106 	riley.jones@nhs.net
+    - 656005750107 	shirley.bryne@nhs.net
+
+  Some valid nhs-login mock usernames are:
+    - 9912003071    for High - P9
```

Comment:
Seems to be mixing up the healthcare worker access mode and the patient access mode which are distinct (check out Access Modes from https://digital.nhs.uk/developer/api-catalogue/personal-demographics-service-fhir )

Reply:
Okay. I've removed the list of nhs-login mock usernames. The other ones make sense for healthcare worker authentication I think.